filament/backend/src/Platform.cpp

```diff
@@ -138,6 +143,8 @@ DefaultPlatform* DefaultPlatform::create(Backend* backend, void* nativeDevice) n
     #if defined(FILAMENT_SUPPORTS_OPENGL)
         #if defined(FILAMENT_USE_EXTERNAL_GLES3) || defined(FILAMENT_USE_SWIFTSHADER)
             return nullptr;
+        #elif defined(FILAMENT_USE_ANGLE)
+            return new PlatformEGL(nativeDevice);
```

Comment:
Please rename the parameter. `nativeDisplay`? And don't forget to update the comment in Platform.h (we forgot to add a description last time)

Reply:
e9c0f52